If TestsFolder contains a space an error is thrown

### Where are you running it?
- Azure DevOps Service (VSTS)

### Version of Extension/Task
Current version

### Expected behaviour and actual behaviour
If the TestFolder variable uses a path that contains a space in it then the Pester test runner will fail. For example my self-hosted agents are installed the the following directory C:\Build Agent\ so when I use $(System.DefaultWorkingDirectory)\* it is evaluated to C:\Build Agent\_work\27\s\\*, this then throws the following error
```
Using executable 'powershell.exe'
powershell.exe C:\Build Agent\_work\_tasks\Pester_cca5462b-887d-4617-bf3f-dcf0d3c622e9\10.2.4\Pester.ps1 -TestFolder C:\Build Agent\_work\26\s\* -resultsFile C:\Build Agent\_work\26\s\Test-Pester.XML -run32Bit False
##[error]C:\Build : The term 'C:\Build' is not recognized as the name of a cmdlet, function, script file, or operable program. 

##[error]C:\Build : The term 'C:\Build' is not recognized as the name of a cmdlet, function, script file, or operable program. 

##[error]Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

### Steps to reproduce the problem
Use a TestsFolder path that contains a space in the path

--- a/Extension/PesterTask/PesterV9/task.json
+++ b/Extension/PesterTask/PesterV9/task.json
@@ -12,7 +12,7 @@
   "author": "Pester",
   "version": {
     "Major": 9,
-    "Minor": 3,
+    "Minor": 4,
     "Patch": 0
   },
   "demands": [],

--- a/Extension/PesterTask/src/pester.ts
+++ b/Extension/PesterTask/src/pester.ts
@@ -92,7 +92,7 @@ export async function run() {
         logInfo(`${executable} ${args.join(" ")}`);
 
         var spawn = require("child_process").spawn, child;
-        child = spawn(executable, args);
+        child = spawn(executable, args, {windowsVerbatimArguments: true});
         child.stdout.on("data", function (data) {
             logInfo(data.toString());
         });

--- a/Extension/PesterTask/src/pesterv10.ts
+++ b/Extension/PesterTask/src/pesterv10.ts
@@ -92,7 +92,7 @@ export async function run() {
         logInfo(`${executable} ${args.join(" ")}`);
 
         var spawn = require("child_process").spawn, child;
-        child = spawn(executable, args);
+        child = spawn(executable, args, {windowsVerbatimArguments: true});
         child.stdout.on("data", function (data) {
             logInfo(data.toString());
         });

